README.md

```diff
@@ -294,11 +301,20 @@ fetcher
     });
 ```
 
-## XHR Params Processing
+## Params Processing
 
-For some applications, there may be a situation where you need to process the service params passed in XHR request before params are sent to the actual service. Typically, you would process these params in the service itself. However, if you want to perform processing across many services (i.e. sanitization for security), then you can use the `paramsProcessor` option.
+For some applications, there may be a situation where you need to
+process the service params passed in the request before they are sent
```

Comment:
Does your editor auto break the lines at a certain character count or is prettier doing this?

Reply:
My editor. Should I undo it?

Reply:
Done ;)